bridge/launchpad/config.go

```diff
@@ -3,25 +3,65 @@ package launchpad
 import (
 	"bufio"
 	"fmt"
+	"net/http"
 	"os"
+	"regexp"
 	"strings"
+	"time"
 
 	"github.com/MichaelMure/git-bug/bridge/core"
 	"github.com/MichaelMure/git-bug/repository"
 )
 
-const keyProject = "project"
+const (
+	keyProject     = "project"
+	defaultTimeout = 60 * time.Second
+)
+
+var (
+	rxLaunchpadURL = regexp.MustCompile(`launchpad\.net[\/:]([^\/]*[a-z]+)`)
```

Comment:
:-O

Reply:
Oops ! will fix that and add necessary tests for launchpad bridge config

Reply:
Done !